itou/employee_record/management/commands/sanitize_employee_records.py

```diff
@@ -37,21 +37,17 @@ def _check_approvals(self, dry_run):
         )
         count_no_approval = no_approval.count()
 
-        self.stdout.write("* Checking missing employee records approval:")
-
         if count_no_approval == 0:
-            self.stdout.write(" - no missing approval (great!)")
+            self.logger.info("no employee records with missing approval (great!)")
         else:
-            self.stdout.write(f" - found {count_no_approval} missing approval(s)")
+            self.logger.info("found %d employee records with missing approval", count_no_approval)
```

Comment:
```suggestion
            self.logger.info("found %d employee records without approval", count_no_approval)
```
Ce qui me ferais presque dire qu'on pourrais juste avoir ce wording et virer le `if`.